### PR TITLE
Docs: Added a guide for use with Nuxt

### PR DESCRIPTION
**Pull request by grbd**
## 📝 Description

Hi, I've been using panda css with nuxt recently.
It seems to work really well so I've added in a guide for it.

If it needs re-wording please let me know

### CODE REVIEW

File: website/pages/docs/getting-started/nuxt.mdx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+```js {1-2, 7-19}  filename="nuxt.config.ts"
+import { createResolver } from '@nuxt/kit'
+const { resolve } = createResolver(import.meta.url)
+
+export default defineNuxtConfig({
+  modules: ['@nuxt/content'],
+
+  alias: {
+    '#pandacss': resolve('./styled-system')
```
Comment (segunadebayo) [inline website/pages/docs/getting-started/nuxt.mdx:141]: Let's keep a consistent naming convention.

```suggestion
    'styled-system': resolve('./styled-system')
```

File: website/pages/docs/getting-started/nuxt.mdx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+  },
+
+  css: [
+    '@/assets/css/panda-css.css',
```
Comment (segunadebayo) [inline website/pages/docs/getting-started/nuxt.mdx:145]: Let's use a more generic name here. Nothing specific to Panda

```suggestion
    '@/assets/css/global.css',
```

File: website/pages/docs/getting-started/nuxt.mdx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+{
+  "scripts": {
+    "prepare": "panda codegen",
+    "build": "pnpm prepare --clean && nuxt build",
+    "dev": "concurrently \"pnpm prepare -w\" \"nuxt dev\"",
+    "generate": "pnpm prepare --clean && nuxt generate",
+    "preview": "nuxt preview"
+  }
+}
+```
```
Comment (segunadebayo) [inline website/pages/docs/getting-started/nuxt.mdx:250]: Let's remove these steps. I don't think it's really Nuxt-related and we have it in the CLI guide already.

**Review by segunadebayo — COMMENTED** [review]
Thanks for working on this. Let some comments

Can you also use https://github.com/chakra-ui/panda/blob/672e13e7d59453d40d4a92f1350d2001b29ad3ed/sandbox/nuxt as a reference for this guide.

**Comment by grbd** [comment]
Ok I've made the changes you've mentioned
I've also altered it to use the bare bones nuxt setup instead of the content module as a starting point
Let me know if you need me to change anything else

**Pull request merged**